src/compiler/program.ts

```diff
@@ -10,7 +10,7 @@ module ts {
     /** The version of the TypeScript compiler release */
     export let version = "1.5.0.0";
 
-    export function createCompilerHost(options: CompilerOptions): CompilerHost {
+    export function createCompilerHost(options: CompilerOptions, setParentNodes?: boolean): CompilerHost {

```

Comment:
I thought about that, but did not feel right, createSourceFile does not know about options. so it will be added, and only honored by the default cmd line host, which is a bit misleading.